services/logs2notifications/internal/handler/useraction_events_template.go

```diff
@@ -0,0 +1,387 @@
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+      </div>
+
+
+      <!--[if mso | IE]></td></tr></table><table align="center" border="0" cellpadding="0" cellspacing="0" class="" role="presentation" style="width:640px;" width="640" bgcolor="#ffffff" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]-->
+
+
+      <div style="-webkit-hyphens: none; -moz-hyphens: none; hyphens: none; word-break: keep-all; background: #ffffff; background-color: #ffffff; margin: 0px auto; max-width: 640px;">
+
+        <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background:#ffffff;background-color:#ffffff;width:100%;">
+          <tbody>
+            <tr>
+              <td style="direction:ltr;font-size:0px;padding:30px 20px 20px 20px;text-align:left;">
+                <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td align="left" class="" style="vertical-align:top;width:20px;" ><![endif]-->
+
+      <div class="mj-column-px-20 mj-outlook-group-fix" style="-webkit-hyphens: none; -moz-hyphens: none; hyphens: none; word-break: keep-all; font-size: 0px; text-align: left; direction: ltr; display: inline-block; vertical-align: top; width: 100%;">
+
+      <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%">
+        <tbody>
+          <tr>
+            <td style="vertical-align:top;padding:0px;">
+
+      <table border="0" cellpadding="0" cellspacing="0" role="presentation" style width="100%">
+        <tbody>
+
+        </tbody>
+      </table>
+
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      </div>
+
+          <!--[if mso | IE]></td><td align="left" class="" style="vertical-align:top;width:580px;" ><![endif]-->
+
+      <div class="mj-column-px-580 mj-outlook-group-fix" style="-webkit-hyphens: none; -moz-hyphens: none; hyphens: none; word-break: keep-all; font-size: 0px; text-align: left; direction: ltr; display: inline-block; vertical-align: top; width: 100%;">
+
+      <table border="0" cellpadding="0" cellspacing="0" role="presentation" width="100%">
+        <tbody>
+          <tr>
+            <td style="vertical-align:top;padding:0px;">
+
+      <table border="0" cellpadding="0" cellspacing="0" role="presentation" style width="100%">
+        <tbody>
+
+              <tr>
+                <td align="left" vertical-align="top" style="font-size:0px;padding:0px;word-break:break-word;">
+
+      <div style="-webkit-hyphens: none; -moz-hyphens: none; hyphens: none; word-break: keep-all; font-family: Montserrat, Arial, sans-serif; font-size: 16px; font-weight: 400; line-height: 21px; text-align: left; color: inherit;"><p class="body_text" style="-webkit-hyphens: none; -moz-hyphens: none; hyphens: none; word-break: keep-all;">
+
+      <!-- This is the main content of the email, which will be rendered in HTML format. -->
```

Comment:
As we discussed, it might be good to separate the main content generation out so that the template is re-usable across the standard email notifications as well as user actions